mpa/cls/trainer.py

```diff
@@ -113,18 +113,22 @@ def run(self, model_cfg, model_ckpt, data_cfg, **kwargs):
         # cfg.dump(osp.join(cfg.work_dir, 'config.yaml')) # FIXME bug to save
         # logger.info(f'Config:\n{cfg.pretty_text}')
 
+        # additional callback
+        save_model_callback = kwargs.get('save_model_callback')
```

Comment:
This is a callback for HPO with OTE and it seems not to be used in MPA, is it correct?

Reply:
yes, you're right. I removed all of save_model_callback related things.